internal/cosmosdb/cosmosdb_client.go

```diff
@@ -0,0 +1,28 @@
+package cosmosdb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	utils "pagopa.it/pagopa-payment-wallet-helpdesk-service/internal/utils"
+)
+
+const MongoConnectionString = "mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@${MONGO_HOSTNAME}:${MONGO_PORT}/${MONGO_DB_NAME}?ssl=${MONGO_SSL_ENABLED}"
+
+func GetMongoClient() (*mongo.Client, error) {
+	uri, err := utils.ReplaceEnvPropertiesInString(MongoConnectionString)
+	if err != nil {
+		errorMessage := fmt.Sprintf("Cannot instantiate Mongo client, error building connection string: [%s]", err)
+		log.Fatal(errorMessage)
+		return nil, errors.New(errorMessage)
+	}
+	additionalMongoConnectionProperties := utils.GetEnvVariableOrDefault("MONGO_ADDITIONAL_CONNECTION_PROPERTIES", "")
+	mongoConnectionString := *uri + additionalMongoConnectionProperties
+	client, err := mongo.Connect(context.TODO(), options.Client().
```

Comment:
From reading the [documentation](https://pkg.go.dev/context#Background) I think we can safely switch this to a background context, wdyt?

```suggestion
	client, err := mongo.Connect(context.Background(), options.Client().
```

Reply:
I was thinking that the context used here should be the one coming from the http request, so here I set the TODO one as a placeholder for future replacing.
I'm refactoring it in order to accept context as input parameter in order for the http request context to be used here as input, what do you think?

Reply:
fixed with https://github.com/pagopa/pagopa-payment-wallet-helpdesk-service/pull/5/commits/67eecf51082db71dea99970b5f804a10a2bf7fe9